paper.tex

```diff
@@ -370,6 +360,35 @@ \section*{Application: simulation using Demes}
     \label{fig:showcase}
 \end{figure}
 
+Here, we highlight the interface between Demes and other software, including
+simulation and model illustration tools. Demes allows us to specify a
+demographic model a single time, which can be used as the input for a growing
+number of simulation software (Table~\ref{tab:software}). We implemented the
+human two-population demographic model from \citet{tennessen2012evolution}
+inferred from European and African-American sequencing data. This model (given
+in Demes format in Figure~\ref{fig:tennessen}) is parameterized by an ancestral
+population with an ancient growth, divergence into ``AFR'' and ``EUR'' that
+each have multiple-epoch size histories, and multiple epochs of continuous
+migration between the two branches (illustrated using \demesdraw\ in
+Figure~\ref{fig:showcase}A). The large final sizes (\(\approx 500,000\)
+individuals each) are one to three orders of magnitude larger than ancestral
+population sizes, reflecting the recent explosive population size increase in
+human populations.
+
+We used this model to simulate 20 haploid genome copies from each EUR and AFR
+at time zero (i.e., present day) to obtain the joint site-frequency spectrum
+(SFS), a summary of observed allele frequencies widely used in evolutionary
+inference
+\citep{bustamante2001directional,gutenkunst2009inferring,tennessen2012evolution,jouganous2017inferring,kamm2017efficient,kim2017inference}.
+We used \msprime\ \citep{baumdicker2021-iu} to simulate a large recombining
+region with the mutation rate assumed in \citet{tennessen2012evolution} and
+computed the observed SFS using \tskit\ \citep{ralph2020efficiently}. We
+compared that simulated SFS to its expectation computed using \moments\
+\citep{jouganous2017inferring} (Figure~\ref{fig:showcase}B,C). For both
+\msprime\ and \moments\ simulations, the same demographic model was used as
```

Comment:
Maybe be more concrete here, "the Demes model shown in Fig XX was used as input"...

Just to hammer home the point that it's precisely this text description that's input.

Reply:
I edited this paragraph quite a bit based on your suggestion. Let me know what you think! Thanks for the suggestions